administration/src/application/components/primitive-inputs/CheckboxForm.tsx

```diff
@@ -22,7 +22,7 @@ const CheckboxForm: Form<CheckboxFormState, Options, ValidatedInput, AdditionalP
     const isInvalid = validationResult.type === 'error'
 
     return (
-      <FormGroup onBlur={() => setTouched(true)}>
```

Comment:
Why did you remove this?

Reply:
because it executed the blur even i click on the Anchor of the data privacy. Dunno why we need this in that case

Reply:
For a general Checkbox, the user should be hinted that it is required to check the checkbox once he moves his focus away from it without checking it. The case with the Datenschutzerklärungs-Dialog is quite special and I think we can live with the hint being shown even if the user only clicked on that button.
So I would propose to revert this small change.